citi-onboarding/Delta-odonto
# Feature/front who we are

## What i did

- Criation the section quem somos

## How to test 

- Go to the directory cliente, using the command in the terminal
`cd client`

- Now use the command bellow in the terminal to starts the Keystone client
` npm start`

- So access in your browser the follow url: localhost: 3000

- You'll can to see the section quem somos with a photo in the left, a text in the right and six icons bellow the text
